docs/cplb.md

```diff
@@ -140,6 +140,7 @@ The Keepalived virtual servers Load Balancing is more performant than the usersp
 1. It's incompatible with controller+worker.
 2. May not work on every infrastructure.
 3. Troubleshooting is significantly more complex.
+4. Requires `sh`, `sed` and `kill` to be present in the control plane nodes.
```

Comment:
I'm a bit reluctant to add more external dependencies when we can avoid it, especially if we want to backport this.

I'd have the following alternative ideas:

* Just write the current state into a file using sh/echo and have a inotify watch on k0s side to get notified about state changes. That'd remove the sed and kill dependencies.
* Call k0s directly and invoke a hidden subcommand that handles all this in Go. That would work in any case without any external dependencies. This would be the only option which would be okay to backport IMO.